eth/catalyst/api.go

```diff
@@ -566,12 +566,31 @@ func (api *ConsensusAPI) getPayload(payloadID engine.PayloadID, full bool) (*eng
 	return data, nil
 }
 
+// GetBlobsV1 returns a blob from the transaction pool.
+func (api *ConsensusAPI) GetBlobsV1(hashes []common.Hash) ([]*engine.BlobAndProofV1, error) {
```

Comment:
Do we want to disable this API if an OP Stack chain is active?

Reply:
I think it's fine to ignore, it's on the engine-API, and no blobs are there, so it's harmless.